src/app/helper/mail-templates/activate_account.html

```diff
@@ -0,0 +1 @@
r:#fff;border-radius:5px;text-align:center}.btn a{background-color:#fff;border:solid 1px #3498db;border-radius:5px;box-sizing:border-box;color:#3498db;cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;padding:12px 25px;text-decoration:none;text-transform:capitalize}.btn-primary table td{background-color:#3498db}.btn-primary a{background-color:#3498db;border-color:#3498db;color:#fff}.last{margin-bottom:0}.first{margin-top:0}.align-center{text-align:center}.align-right{text-align:right}.align-left{text-align:left}.clear{clear:both}.mt0{margin-top:0}.mb0{margin-bottom:0}hr{border:0;border-bottom:1px solid #f6f6f6;Margin:20px 0}@media only screen and (max-width:620px){table[class=body] h1{font-size:28px!important;margin-bottom:10px!important}table[class=body] a,table[class=body] ol,table[class=body] p,table[class=body] span,table[class=body] td,table[class=body] ul{font-size:16px!important}table[class=body] .article,table[class=body] .wrapper{padding:10px!important}table[class=body] .content{padding:0!important}table[class=body] .container{padding:0!important;width:100%!important}table[class=body] .main{border-left-width:0!important;border-radius:0!important;border-right-width:0!important}table[class=body] .btn table{width:100%!important}table[class=body] .btn a{width:100%!important}table[class=body] .img-responsive{height:auto!important;max-width:100%!important;width:auto!important}}@media all{.ExternalClass{width:100%}.ExternalClass,.ExternalClass div,.ExternalClass font,.ExternalClass p,.ExternalClass span,.ExternalClass td{line-height:100%}.apple-link a{color:inherit!important;font-family:inherit!important;font-size:inherit!important;font-weight:inherit!important;line-height:inherit!important;text-decoration:none!important}.btn-primary table td:hover{background-color:#34495e!important}.btn-primary a:hover{background-color:#34495e!important;border-color:#34495e!important}}</style><table border=0 cellpadding=0 cellspacing=0 class=body><tr><td>&nbsp;<td class=container><div class=content><table class=main><tr><td class=wrapper><table border=0 cellpadding=0 cellspacing=0><tr><td><table border=0 cellpadding=0 cellspacing=0 class=btn><tr><td align=center><table border=0 cellpadding=0 cellspacing=0><tr><td><img src=https://share.sirimangalo.org/logo_mail.png></table></table><p>Welcome %{{newUser}}%,<p>in order to activate your new account, please verify your email address by clicking on the link below.<table border=0 cellpadding=0 cellspacing=0 class="btn btn-primary"><tr><td align=center><table border=0 cellpadding=0 cellspacing=0><tr><td><a href=%{{activationLink}}% target=_blank>Activate Account</a></table></table>If you have problems logging in or technical issues, please send an email to <a href=mailto:it@sirimangalo.org>it@sirimangalo.org</a>.</table></table><div class=footer><table border=0 cellpadding=0 cellspacing=0><tr><td class=content-block>This email was automatically generated</table></div></div><td>&nbsp;</table>
```

Comment:
No that's negligible. It's more important to have a html file which is easy to maintain. What's the difference between this file an `email-mockup.html`?